sdk/include/aditof/camera_definitions.h

```diff
@@ -30,6 +31,33 @@ enum class ConnectionType {
     LOCAL     //!< The device is embedded in the system
 };
 
+/**
+ * @struct IntrinsicParameters
+ * @brief Provides the intrinsic parameters of a camera.
```

Comment:
'Describes' is better suited than 'Provides' since there is no action done by this structure. Maybe fix the description of the ConnectionType as well.

Reply:
solved in commit https://github.com/analogdevicesinc/aditof_sdk/pull/203/commits/adba4901642e22b8626a1910e3c23271c2af6dc3